terraform/keycloak-dev/realms/onestopauth/client-work-bc-job-board-496.tf

```diff
@@ -6,4 +6,5 @@ module "client_work-bc-job-board-496" {
     "https://dev-admin-jobboard.workbc.ca/*",
     "https://localhost:44364/*"
   ]
+  web_origins           = ["https://dev-admin-jobboard.workbc.ca, https://localhost:44364"]
```

Comment:
Hey, I was just looking at the original client definition. I think that it expects an array of strings for web_origins istead of one string, from [here](https://registry.terraform.io/providers/mrparkers/keycloak/latest/docs/resources/openid_client#web_origins) the type is the same as `valid_redirect_uris`

Reply:
I mean one string per url instead of a comma separated string

Reply:
Thanks @jlangy - pushed an update.